### DISCUSSION TITLE
[discussion] Header 공통 스타일 관리

### DISCUSSION THREAD

**Discussion by July249** [beyondDevelops/AnimalTalk#29] [Q&A] [answered]
Header에게 들어가는 스타일 중 공통되는 것들을 index.css에서 클래스로 묶어 스타일 관리를 용이하게 하는 것은 어떨까요?

Originally posted by @July249 in https://github.com/beyondDevelops/AnimalTalk/pull/23#discussion_r1047220107

**Comment by GreattitJY** [comment, marked as answer]
페이지 전체에서 전역으로 쓰이면 index.css에서 묶는 게 좋지만 하나의 컴포넌트에서만 사용된다면 따로 빼지 않아도 될 거 같습니다!

**Comment by July249** [comment]
하기의 내용은 이미 회의 때 언급된 사안입니다. 기록이 누락되어 작성하였습니다.

---

리액트 자체의 컴포넌트 재활용성이 높다는 이점과 TailwindCSS의 가장 큰 이점인 HTML 구조 작성과 동시에 스타일링이 가능하다는 점을 고려하면 굳이 공통 스타일을 관리하는 컴포넌트를 만들어 스타일 관리를 이중으로 하는 것은 TailwindCSS가 주는 이점을 포기하는 것과 같다는 판단이 들었습니다.

따라서 버튼 스타일과 레이아웃 스타일과 같이 재사용성이 아주 높은 경우가 아니라면 index.css에서 관리할 필요성이 없다고 판단되었습니다.
